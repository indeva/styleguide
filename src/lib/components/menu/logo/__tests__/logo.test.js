import React from 'react';
import renderer from 'react-test-renderer';
import Logo from '..';

describe('Logo menu component', () => {
  it('should render properly when is not minified', () => {
    const component = renderer.create(<Logo collapsed={false} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render properly when is minified', () => {
    const component = renderer.create(<Logo collapsed />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
