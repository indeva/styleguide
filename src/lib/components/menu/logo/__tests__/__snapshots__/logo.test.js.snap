// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo menu component should render properly when is minified 1`] = `
<div
  className="menu__logo-wrapper menu__logo-wrapper--collapsed"
>
  <img
    className="menu__logo topbar__logo--in"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    src="logo-in.svg"
  />
  <img
    className="menu__logo topbar__logo--deva"
    src="logo-deva.svg"
  />
</div>
`;

exports[`Logo menu component should render properly when is not minified 1`] = `
<div
  className="menu__logo-wrapper"
>
  <img
    className="menu__logo topbar__logo--in"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    src="logo-in.svg"
  />
  <img
    className="menu__logo topbar__logo--deva"
    src="logo-deva.svg"
  />
</div>
`;
